docs/source/getting-started/content/proc_troubleshooting-bundle-version-mismatch.adoc

```diff
@@ -0,0 +1,31 @@
+[id="troubleshooting-bundle-version-mismatch_{context}"]
+= Troubleshooting bundle version mismatch
+
+As {prod} releases are set up and started, instances are created with bundle information and instance data.  When a new {prod} release is setup, the bundle information and instance data is not updated due to the customization in the previous instance data.  This will lead to errors on [command]`{bin} start` command. The error will be similar to this:
```

Comment:
Could you please put each sentence on its own line?

Reply:
Moved sentences to their own line.